lib/src/widgets/overlay.dart

```diff
@@ -599,7 +601,7 @@ class _DescribedFeatureOverlayState extends State<DescribedFeatureOverlay> with
               child: _Background(
                 transitionProgress: _transitionProgress,
                 color: widget.backgroundColor ?? Theme.of(context).primaryColor,
-                defaultOpacity: widget.backgroundOpacity ?? 0.96,
```

Comment:
@BazinC It's already (implicitly) a constant when you set it as a default parameter, because non-constant default parameters are not compilable.